user-manual/access-content/search-atom.rst

```diff
@@ -779,6 +783,22 @@ with particular fields in the authority record edit template, see:
    the results, and AtoM will redirect you to the selected authority record's
    :term:`view page`.
 
+**To search for authority records in AtoM using the advanced search options:**
```

Comment:
At this point you can probably leave this here, but I would have kept this separate, and added mention of it and links to the full docs  in Advanced search as part of a separate PR, when we work on the Authority record advanced search documentation.

Reply:
I considered that but decided to put the instructions on this page because there are similar instructions here for repository searching.